components/bl-parallax-component/src/index.js

```diff
@@ -1,15 +1,25 @@
-import { useRef, useEffect, useCallback } from 'react';
+import { useRef, useEffect, useCallback, useState } from 'react';
 
 const { cn } = BackendlessUI.CSSUtils;
 
 export default function Parallax({ component, pods }) {
   const { display, style, classList, imageUrl, strength } = component;
   const parallaxContentPod = pods['parallaxContent'];
 
+  const [power, setPower] = useState(0);
+
   const backdropRef = useRef();
   const containerRef = useRef();
 
-  useAnimation(backdropRef, containerRef, strength);
+  useEffect(() => {
+    if (typeof strength === 'number') {
+      setPower(strength)
+    } else {
+      setPower(0)
+    }
```

Comment:
```suggestion
      setPower(typeof strength === 'number' ? strength : 0)
```

And if we already use strength - no need in one more definition for it. Use `validStrength` for state variable.